### PR TITLE
SSCS-8954:Maternity Allowance (MYA)- Update: MYA Landing page

### PR DESCRIPTION
**Pull request by rasikasri**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-8954

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by chrisdavecm — APPROVED** [review]

File: locale/content.json [hunk 1]
```diff
@@ -428,7 +428,8 @@
          "attendanceallowance": "Return to your Attendance Allowance appeal",
          "bereavementbenefit": "Return to your Bereavement Benefit appeal",
          "jsa": "Return to your JSA Benefit appeal",
-         "industrialinjuriesdisablement": "Return to your Industrial Injuries Disablement Benefit appeal"
+         "industrialinjuriesdisablement": "Return to your Industrial Injuries Disablement Benefit appeal",
+          "maternityallowance": "Return to your Your Maternity Allowance appeal"
```
Comment (danielbe11) [inline locale/content.json:432]: "your Your" typo

File: locale/content.json [hunk 2]
```diff
@@ -802,6 +803,14 @@
         "agency": "Department for Work and Pensions",
         "agencyAcronym": "DWP",
         "panel": "a judge"
+      },
+      "maternityallowance" : {
+        "acronym": "Maternity Allowance",
```
Comment (danielbe11) [inline locale/content.json:808]: Should be "MATERNITY ALLOWANCE" to match other acronyms above.

File: locale/content.json [hunk 3]
```diff
@@ -802,6 +803,14 @@
...
+        "yourBenefitAppeal": "Your Maternity Allowance appeal",
+        "fullDescription": "Maternity Allowance",
+        "agency": "Department for Work and Pensions",
+        "agencyAcronym": "DWP",
+        "panel": "a judge and a doctor"
```
Comment (danielbe11) [inline locale/content.json:813]: Should be "a judge" according to https://tools.hmcts.net/jira/browse/SSCS-8956

File: locale/content.json [hunk 4]
```diff
@@ -1803,6 +1813,14 @@
...
         "agencyAcronym": "DWP",
         "panel": "barnwr"
+      },
+      "maternityallowance": {
+        "acronym": "LWFANS MAMOLEATH",
+        "yourBenefitAppeal": "Eich apêl Lwfans Mamolaeth",
+        "fullDescription": "Lwfans Mamolaeth",
+        "agency": "Yr Adran Gwaith a Phensiynau",
+        "agencyAcronym": "DWP",
+        "panel": "barnwr a meddyg"
```
Comment (danielbe11) [inline locale/content.json:1823]: Should be "barnwr" according to https://tools.hmcts.net/jira/browse/SSCS-8956

**Pull request merged**